release-team/role-handbooks/emeritus-adviser/README.md

```diff
@@ -2,24 +2,35 @@
 
 ## Overview
 
-The Emeritus Adviser (EA) is an experienced Release Team (RT) member who returns to the team in an advisory role.
+The Emeritus Adviser (EA) is an experienced Release Team (RT) member who returns to the team in an advisory role.  
 
-## Responsibilities
+### Requirements and Selection
+The EA should be an individual who has served on the release team, in a lead capacity.  The primary selection pool should be former release leads (ideally from the previous 4 releases), however if there are no available individuals the EA can come from other former role leads who have been an exemplary member of the previous release teams.  
 
-The primary duty of the EA is
-managing personnel for the Release Team, including Role Leads and especially Shadows.  The EA monitors the Shadow program and making sure that it is working as an effective apprenticeship process.
+An ideal Emeritus Adviser is inherently suited to people management, has shown sustained interest in the health and growth of SIG Release, and has a commitment to Release Team diversity and helping others ascend the [Contributor Ladder](https://git.k8s.io/community/community-membership.md).
 
-As a returning RT member, the EA is also there to supply the Release Lead with historical information about how things were handled in prior releases, and serve as a sounding board when both the Release Lead and Lead Shadows are facing a dilemma.
+Time commitments for the EA are relatively light, with the largest time commitment in the two-week period where Shadows are being selected.  However, the EA role starts 2-3 weeks before the release cycle officially begins, and continues for 1-2 weeks after it ends.
 
-## Requirements
+EAs are not chosen as part of the regular RT selection process.  Instead, interested candidates should contact the outgoing EA and the incoming Release Lead 2 weeks before the final prior release.  Most of the time, the outgoing EA will solicit suitable candidates.
```

Comment:
We need to tweak this based on our discussion.
I think this can be an Emeritus Adviser's _suggestion_, but the _selection_ should be made by the outgoing and incoming leads (to make sure it's inclusive).

Reply:
just to clarify this is talking about the selection of the EA role (not the release team lead succession) 

though if that is unclear can put in more clear wording that this is for EA succession

Reply:
Yeah, this needs a tweak as we've tapped people for this role in previous cycles.
I think the outgoing Emeritus Adviser should put forth a set of candidates which are then selected from the outgoing and incoming Release Team Leads.

Reply:
to clarify my understanding the suggestion is that hypothetically for this cycle I would have nominated 2-3 people and had Guin and Jorge select an individual to be EA for 1.18?

Reply:
Yep!

Reply:
updated